web/controllers/admin.rb

```diff
@@ -1,6 +1,7 @@
 module Web; end
 require "web/helpers"
 require "lib/admin_csv_generator"
+require 'sinatra'
```

Comment:
Again I'm not sure what is this `require` for exactly

Reply:
Same as above, fixes a warning about things being explicitly required for me